README.md

```diff
@@ -104,6 +104,14 @@ brew install cmake openssl@1.1
 You can install `cmake`, `clang`, and `openssl` with your distro's package manager or download them from their websites.
 On Debian and Ubuntu, you will also need the `build-essential` and `libudev-dev` packages.
 
+### Testing
+
+When working on the SDK itself, run its tests using `cargo` with `storage` feature enabled:
```

Comment:
Not sure about this, I mean tests should also work fine without enabling this feature

Reply:
But then it should rather suggest testing with --all-features, no? Because we have more features not enabled by default than the storage (also rocksdb has an own feature, so this is also different to storage alone)

Reply:
Why not just suggest running what the CI runs, so `cargo ci-test`?

Reply:
I think I agree with Thoralf, we should suggest `-all-features`. And if there has been a gap in the feature set, it's a CI issue that we need to fix.

Reply:
I think it's just confusing to locally run something that is called CI. We could rename it idk 🤷🏻‍♂️

Reply:
But IMO I'd just give the whole command and be explicit